arangod/Replication/Syncer.h

```diff
@@ -121,6 +121,22 @@ class Syncer : public std::enable_shared_from_this<Syncer> {
     /// response was received or if something went wrong)
     arangodb::Result _res;
 
+    /// @brief the callback to execute
+    std::function<void()> _cb;
```

Comment:
fu2 unique_function?

Reply:
Ok, changed to fu2::unique_function<...>